bears/python/PycodestyleBear.py

```diff
@@ -1,6 +1,6 @@
 from coalib.bearlib.abstractions.Linter import linter
-
```

Comment:
please do not remove this blank line.  That is not part of https://github.com/coala/coala-bears/issues/1739 , so dont do it.